subsys/bluetooth/controller/Kconfig

```diff
@@ -112,6 +112,22 @@ config BLUETOOTH_CONTROLLER_LE_PING
 	help
 	  Enable support for Bluetooth v4.1 LE Ping feature in the Controller.
 
+config BLUETOOTH_CONTROLLER_PRIVACY
+	bool "LE Controller-based Privacy"
+	default y
+	select BLUETOOTH_RPA
+	help
+	  Enable support for Bluetooth v4.2 LE Controller-based Privacy feature in the Controller.
+
+config BLUETOOTH_CONTROLLER_RL_SIZE
+	prompt "LE Controller-based Privacy Resolving List size"
+	depends on BLUETOOTH_CONTROLLER_PRIVACY
+	int
+	default 8
+	range 1 8
```

Comment:
Why limit the max to 8? I wonder if we should just use the existing CONFIG_BLUETOOTH_MAX_PAIRED instead?